src/UserAccountDataChecker.php

```diff
@@ -24,13 +24,13 @@ class UserAccountDataChecker extends DataChecker {
 	public $mUser = null;
 
 	public function __construct() {
-		$this->addCheck( array( &$this, 'checkDomainValidity' ), array() );
-		$this->addCheck( array( &$this, 'checkDomainUser' ), array() );
-		$this->addCheck( array( &$this, 'checkCreatePermissions' ), array() );
-		$this->addCheck( array( &$this, 'checkSorbs' ), array() );
-		$this->addCheck( array( &$this, 'checkUserExists' ), array() );
-		$this->addCheck( array( &$this, 'checkPasswordLength' ), array() );
-		$this->addCheck( array( &$this, 'checkEmailValidity' ), array() );
+		$this->addCheck( array( $this, 'checkDomainValidity' ), array() );
```

Comment:
So it should be
```
$this = $checkDomainValidity
$this->addCheck( array( $checkDomainValidity, 'checkDomainValidity' ), array() );
```
instead of 
```
$this->addCheck( array( $this, 'checkDomainValidity' ), array() );
```
And this seven times within the function?